blueprints/component/index.js

```diff
@@ -32,20 +32,23 @@ module.exports = {
     },
 
     locals(options) {
-        let importStyles = '';
-        let importTemplate = '';
-        let contents = '';
+        let osfImports = "import { localClassNames } from 'ember-osf-web/decorators/css-modules';";
```

Comment:
I've actually been thinking we shouldn't add this automatically in the blueprint, especially since we are moving toward glimmer components.